cmd/cli/plugin/package/package_available_list.go

```diff
@@ -92,3 +105,25 @@ func packageAvailableList(cmd *cobra.Command, args []string) error {
 	t.RenderWithSpinner()
 	return nil
 }
+
+// getPackageLatestVersion returns the latest version of a particular package under a namespace
+func getPackageLatestVersion(packageName, namespace string, kc kappclient.Client) (string, error) {
+	pkgs, err := kc.ListPackages(packageName, namespace)
+	if err != nil {
+		return "", err
+	}
+	latest := ""
+	for i := range pkgs.Items {
+		pkg := pkgs.Items[i]
+		current := strings.ToLower(pkg.Spec.Version)
+		if latest == "" {
+			latest = current
+		} else if strings.Compare(latest, current) < 0 {
```

Comment:
I should put the comment at L122 more clear. What I meant was `semver.Compare`(used in other places) from https://pkg.go.dev/golang.org/x/mod/semver only accepts `vMAJOR[.MINOR[.PATCH[-PRERELEASE][+BUILD]]]`. So that I have to use `strings.Compare` which works in our case but might not be a ideal way to compare versions.